ext/native-decls/GetPlayerFromServerId.md

```diff
@@ -10,6 +10,7 @@ Player GET_PLAYER_FROM_SERVER_ID(int serverId);
 
 
 ## Parameters
-* **serverId**: 
+* **serverId**: the players network id
```

Comment:
the players (plural of player?) network id? hm?

Reply:
Yea shouldn't be plural 😆 Regarding network id you refer to this as the NetID in other parts of the scripting manual [like here](https://docs.fivem.net/docs/scripting-manual/networking/ids/), would you like me to just leave it as `the player's server id`?